erpnext/loan_management/doctype/loan_repayment/loan_repayment.py

```diff
@@ -63,6 +65,13 @@ def set_missing_values(self, amounts):
 		if amounts.get('due_date'):
 			self.due_date = amounts.get('due_date')
 
+	def check_future_entries(self):
+		future_repayment_date = frappe.db.get_value("Loan Repayment", {"posting_date": (">", self.posting_date),
+			"docstatus": 1}, 'posting_date')
+
+		if future_repayment_date:
```

Comment:
(Minor) You can use `frappe.db.exists` here. Also, no need to request `posting_date` specifically. The default `name` can work too.

Reply:
I am using the posting_date in the below error message, hence queried posting_date

Reply:
Sorry, missed that